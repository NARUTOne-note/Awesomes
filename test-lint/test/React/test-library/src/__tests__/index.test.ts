import * as bdx from '../index';

describe('bdx', () => {
  test('exports modules correctly', () => {
    expect(Object.keys(bdx)).toMatchSnapshot();
  });
});
