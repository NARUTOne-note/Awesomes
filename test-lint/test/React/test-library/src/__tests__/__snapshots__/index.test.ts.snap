// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`bdx exports modules correctly 1`] = `
Array [
  "SpanEdit",
  "SortArrow",
  "TitleTag",
]
`;
